migrate/20231013_dns_servers.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:dns_server) do
+      column :id, :uuid, primary_key: true, default: nil
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :name, :text, collate: '"C"', null: false, unique: true
+    end
+
+    create_table(:dns_servers_dns_zones) do
```

Comment:
Why do we use plural names for pivot tables? Is is required by `Sequel`?

Reply:
yes, this is the default for :join_table
>name of table that includes the foreign keys to both the current model and the associated model, as a symbol. Defaults to the name of current model and name of associated model, pluralized, underscored, sorted, and joined with ‘_’.